.github/ISSUE_TEMPLATE/default-new.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Issue
```

Comment:
*comment:* 

@Marie-L in regards to the tag there's a `tag` you can use here to automatically add any desired one to issues upon creation. CC: @albendz

Reply:
Are there docs on this? I can't find anything after searching for a while

Reply:
@lorenanicole Ah ok, thanks for letting me know

Reply:
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/4573607/74274548-dc8b0780-4cd7-11ea-8310-aca4db2203c7.png">

It's the `labels` - https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository. Sorry brain hiccup and called it tag, heh.